lib/cn/omf.h

```diff
@@ -152,17 +152,14 @@ OMF_SETGET(struct bloom_hdr_omf, bh_n_hashes, 8)
  *         version even though the actual WBTree header, leaf and
  *         internal nodes are no different from OMF v5.
  *
- * OMF v5: Added longest common prefix elimination for keys in WBTree nodes.
- *
- * OMF v4: Added new value, "immediate", for short values.
- *
- * OMF v3 and below: Ancient history. Nobody cares. Should remove support.
+ * OMF v5: Added longest common prefix elimination for keys in WBTree nodes (unsupported).
+ * OMF v4: Added new value, "immediate", for short values (unsupported).
```

Comment:
Odd to document v4 and v5, which are no longer supported, and not document v6.   Might dig up what changed between v5 and v6 and change comment to:
```
Supported versions:
   v6: ...
Deprecated versions:
  v5: ...
```

Reply:
Fixed. The comment for what changed between v5 and v6 already exists above.